ironfish-cli/src/utils/longPrompt.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import readline from 'readline'
+
+// Most effective way to take in a large textual prompt input without affecting UX
```

Comment:
I'm wondering if this is fixing a MacOS-specific problem? If so, I would write it in the comment, otherwise someone like me might come around and remove this function if they cannot reproduce the lag problem